php-classes/PasswordAuthenticator.class.php

```diff
@@ -41,6 +43,16 @@ public function checkAuthentication()
             $this->_authenticatedPerson = $this->attemptAuthentication($requestData['username'], $requestData['password']);
 
             if ($this->_authenticatedPerson) {
+                
+                // set cookie for username
+                if (static::$enableRememberMe && isset($requestData['remember'])) {
+                    setCookie(static::$rememberCookie, $requestData['username'], time() + static::$rememberDuration);
+                    
+                // otherwise destroy if active
+                } elseif (static::$enableRememberMe) {
+                    setcookie(static::$rememberCookie, "", time() - 3600);
```

Comment:
@jtwiest use single quotes for strings where interpolation isn't being used for consistency (and ever so slight performance)